chromium/debugging-rulesets.html

```diff
@@ -4,20 +4,21 @@
     <meta charset="utf-8">
     <title></title>
     <link href="debugging-rulesets.css" rel="stylesheet">
+    <script src="external/codemirror-5.31.0.min.js"></script>
+    <link href="external/codemirror-5.31.0.min.css" rel="stylesheet">
+    <script src="external/codemirror-5.31.0.xml.min.js"></script>
```

Comment:
I may have not been specific enough in my suggestion.  Can we move each of these under `external/codemirror/`, just so each dependency moving forward is in its own neat path?

Reply:
Done.